modules/gdnative/gdnative_api.json

```diff
@@ -11,5747 +11,9595 @@
         "name": "godot_color_new_rgba",
         "return_type": "void",
         "arguments": [
-          ["godot_color *", "r_dest"],
-          ["const godot_real", "p_r"],
-          ["const godot_real", "p_g"],
-          ["const godot_real", "p_b"],
-          ["const godot_real", "p_a"]
+          [
+            "godot_color *",
+            "r_dest"
+          ],
```

Comment:
Seems like you formatted a good chunk of the file, can you revert that and add your changes in the same style as the rest?

Reply:
Sigh. vscode autoformats every time I save the file. I'll change it.

Reply:
Boop comment doesn't exist in my file, and I can't even access the file. It's weird.